tests/unit/amo/components/TestAddonMeta.js

```diff
@@ -53,6 +65,31 @@ describe('<AddonMeta>', () => {
       });
       expect(getUserCount(root)).toMatch(/^1\.000/);
     });
+
+    it('links to stats if add-on author is viewing the page', () => {
+      const addon = {
+        ...fakeAddon,
+        slug: 'coolio',
+        authors: [
+          {
+            id: 11,
+            name: 'tofumatt',
+            picture_url: 'http://cdn.a.m.o/myphoto.jpg',
+            url: 'http://a.m.o/en-GB/firefox/user/tofumatt/',
+            username: 'tofumatt',
+          },
+        ],
+      };
+      const root = render({
+        addon,
+        store: dispatchSignInActions({ userId: 11 }).store,
```

Comment:
instead of using a magic number, this should reference a variable for the `userId` value.